http_client.rst

```diff
@@ -125,24 +146,107 @@ Authentication
 ~~~~~~~~~~~~~~
 
 The HTTP client supports different authentication mechanisms. They can be
-defined globally when creating the client (to apply it to all requests) and to
-each request (which overrides any global authentication)::
+defined globally in the configuration (to apply it to all requests) and to
+each request (which overrides any global authentication).
+
+Use the ``scoped_clients`` options to create an HTTP client service with
+pre-configured options. Read more about them :ref:`below <http-scoped-clients>`:
+
+.. configuration-block::
+
+    .. code-block:: yaml
+
+        # config/packages/framework.yaml
+        framework:
+            http_client:
+                scoped_clients:
+                    # This creates a new HTTP Client service named "example_api".
+                    # It will be autowired in arguments type-hinting "HttpClientInterface $example_api"
```

Comment:
For PHP code we always use variables such as `$exampleApi` instead of `$example_api`. We should consider upgrading this name.